examples/php/composer.json

```diff
@@ -7,7 +7,9 @@
   },
   "autoload": {
     "psr-4": {
-      "": "route_guide/"
+      "": "route_guide/",
```

Comment:
Can you see if you can fix this line as well? Instead of a wildcard (everything) class matching to the `route_guide/` directory, can you explicitly mark them as `"Routeguide\\": "route_guide/"? The code is here: https://github.com/grpc/grpc/blob/master/examples/php/route_guide/route_guide_client.php

But in doing so, can you please make sure https://grpc.io/docs/tutorials/basic/php/ is still runnable correctly? Thanks.

Reply:
I have verified, it's working fine. still need to keep "GPBMetadata" line.